target/linux/ramips/dts/ArcherC2.dts

```diff
@@ -0,0 +1,168 @@
+/dts-v1/;
+
+/include/ "mt7620a.dtsi"
+
```

Comment:
Please include `dt-bindings/gpio/gpio.h` and `dt-bindings/input/input.h` as all other dts files are doing.

Use the GPIO_ACTIVE_LOW and GPIO_ACTIVE_HIGH macros afterwards in stead of 1 and 0 in the gpio parameters.

Same for the [key code macros](https://elixir.bootlin.com/linux/v4.14.41/source/include/uapi/linux/input-event-codes.h#L572) instead of the hex values.

Check the recent [ramips board additions](https://github.com/lede-project/source/commits/master/target/linux/ramips) for examples.